src/Client/Core/DurableTaskClient.cs

```diff
@@ -337,7 +337,7 @@ public virtual Task ResumeInstanceAsync(string instanceId, CancellationToken can
     public abstract AsyncPageable<OrchestrationMetadata> GetAllInstancesAsync(OrchestrationQuery? filter = null);
 
     /// <inheritdoc cref="PurgeInstanceAsync(string, PurgeInstanceOptions, CancellationToken)"/>
-    public virtual Task<PurgeResult> PurgeInstancesAsync(string instanceId, CancellationToken cancellation)
+    public virtual Task<PurgeResult> PurgeInstancesAsync(string instanceId, CancellationToken cancellation = default)
```

Comment:
From a binary compatibility perspective, this is still a breaking change. We need to introduce a virtual method that doesn't include the `CancellationToken` parameter at all.

```csharp
public virtual Task<PurgeResult> PurgeInstancesAsync(string instanceId, CancellationToken cancellation)
    => this.PurgeInstanceAsync(instanceId, CancellationToken.None);
```

Reply:
Wait, maybe my suggestion isn't the right one. I'd need to double check where the failure is coming from.

Reply:
I'll be honest, I haven't had the chance to validate my diff yet either. Just FYI